### PR TITLE
A converter from jsonld.

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: converters/jsonldToManifest.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    s += '\n';
+
+    return s;
+  }
+}
+
+module.exports = JsonldToManifest;
+
+var fs = require('fs');
+console.log(JsonldToManifest.convert(fs.readFileSync("C:/Users/shanestephens/Downloads/Product.jsonld", "utf-8")));
```
Comment (dstockwell) [inline converters/jsonldToManifest.js]: remove this

File: converters/jsonldToManifest.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        relevantProperties.push({name, type});
+      }
+    }
+
+    var className = theClass['@id'].split(':')[1];
+    var superName = theClass.superclass ? theClass.superclass['@id'].split(':')[1] : null;
+
+    var s = '';
+    if (superName !== null)
+      s += `import 'https://schema.org/${superName}'\n\n`
```
Comment (dstockwell) [inline converters/jsonldToManifest.js]: should we pass in the base URL or a resolver function as a param? they might not always come from schema.org.

Comment (shans) [inline converters/jsonldToManifest.js]: I think not until we understand what's going on with extension schemas

File: converters/jsonldToManifest.js (hunk 1 above)
Comment (shans) [inline converters/jsonldToManifest.js]: done

**Pull request merged**